src/sentry/relay/config/__init__.py

```diff
@@ -213,47 +185,107 @@ def _get_project_config(project, full_config=True, project_keys=None):
                         )
                     active_rules.append(rule)
 
-            cfg["config"]["dynamicSampling"] = {"rules": active_rules}
+            return {"rules": active_rules}
+
+    return None
+
+
+def add_experimental_config(config: dict, key: str, function, *args, **kwargs):
```

Comment:
missing annotation for function?

Reply:
should be `Callable` https://docs.python.org/3/library/typing.html#typing.Callable

Reply:
This module isn't really typed, but here we go:

https://github.com/getsentry/sentry/blob/350adbf250be7fa7e18fd556f964cfd6812acb8e/src/sentry/relay/config/__init__.py#L203-L209